### PR TITLE
Adds a pilotable whiteship to Deltastation

### PR DESCRIPTION
**Pull request by ghost**
[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl: Mark9013100
add: Deltastation now has a Whiteship.
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)

This adds a decently sized Whiteship to deltastation, it contains a decent amount of loot similar to Meta's whiteship. This brings it in line with the other maps as it was the only medium/highpop map without one. Massive thanks to Wjohn and Okand for assistance with the technical side of things.

Also adds a 50 stack variant of titanium sheets, one is mapped on the ship for construction purposes.

Screenshots:
dreammaker_2017-11-04_10-43-49
dreammaker_2017-11-04_10-44-18

(It isn't aligned like that ingame, I just got lazy and made a quick capture.)

### CODE REVIEW

File: _maps/shuttles/whiteship_delta.dmm [hunk 1]
```diff
@@ -0,0 +1,213 @@
+"aa" = (/turf/open/space/basic,/area/space)
```
Comment (ShizCalev) [inline _maps/shuttles/whiteship_delta.dmm]: TGM.

**Review by ShizCalev — DISMISSED** [review]

File: _maps/shuttles/whiteship_delta.dmm (hunk 1 above)
Comment (Mark9013100) [reply _maps/shuttles/whiteship_delta.dmm]: I did use TGM, though I fix that within the hour when I get the chance.

**Comment by GunHog** [comment]
Pretty little ship!

**Comment by praisenarsie** [comment]
What's the PDA in the bridge?

**Comment by Mark9013100** [comment]
It's the rare transparent case PDA.

**Comment by davethwave** [comment]
Did you put a GPS in it?

**Comment by Mark9013100** [comment]
Yes, @davethwave

**Comment by Ratvar** [comment]
make it alligned like this ingame

**Comment by Okand37** [comment]
Looks cute! I can't quite see an actual showerhead in the shower, however!

**Comment by Mark9013100** [comment]
It's there, just need to use alt+click to be able to use it.

**Comment by WJohn** [comment]
that's not terribly good to do that. have it propped up against the north wall so people can see it. you'll need a smidge of pixel shifting to do it. i also noticed you didn't pixel shift the toilet, you'll want to do that.

**Comment by Mark9013100** [comment]
Alright, I'll fix that in a few, any suggestions on an offset?

**Comment by ninjanomnom** [comment]
If you want to you can make the ship have multiple areas and a powernet.

**Comment by Mark9013100** [comment]
I'd prefer to do that in a future PR, multiple areas I'm a bit iffy on, but giving it a powernet is something I'll definitely want to do.

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 2]
```diff
@@ -102607,6 +102607,2890 @@
...
+/obj/effect/decal/cleanable/dirt{
+	desc = "A thin layer of dust coating the floor.";
+	name = "dust"
+	},
+/turf/open/floor/plasteel/neutral/side{
+	dir = 1
+	},
+/area/shuttle/abandoned)
+"YNb" = (
+/obj/structure/closet/emcloset{
```
Comment (ShizCalev) [inline _maps/map_files/Deltastation/DeltaStation2.dmm]: /obj/structure/closet/emcloset/anchored

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 3]
```diff
@@ -102607,6 +102607,2890 @@
...
+/obj/effect/decal/cleanable/robot_debris,
+/turf/open/floor/plasteel/neutral/side,
+/area/shuttle/abandoned)
+"YKw" = (
+/obj/effect/decal/cleanable/dirt{
+	desc = "A thin layer of dust coating the floor.";
+	name = "dust"
+	},
+/obj/effect/decal/cleanable/cobweb/cobweb2,
+/obj/structure/closet/emcloset{
```
Comment (ShizCalev) [inline _maps/map_files/Deltastation/DeltaStation2.dmm]: /obj/structure/closet/emcloset/anchored

File: _maps/shuttles/whiteship_delta.dmm [hunk 4]
```diff
@@ -0,0 +1,2809 @@
...
+/obj/effect/decal/cleanable/robot_debris,
+/turf/open/floor/plasteel/neutral/side,
+/area/shuttle/abandoned)
+"bH" = (
+/obj/effect/decal/cleanable/dirt{
+	desc = "A thin layer of dust coating the floor.";
+	name = "dust"
+	},
+/obj/effect/decal/cleanable/cobweb/cobweb2,
+/obj/structure/closet/emcloset{
```
Comment (ShizCalev) [inline _maps/shuttles/whiteship_delta.dmm]: /obj/structure/closet/emcloset/anchored

File: _maps/shuttles/whiteship_delta.dmm [hunk 5]
```diff
@@ -0,0 +1,2809 @@
...
+/obj/effect/decal/cleanable/dirt{
+	desc = "A thin layer of dust coating the floor.";
+	name = "dust"
+	},
+/turf/open/floor/plasteel/neutral/side{
+	dir = 1
+	},
+/area/shuttle/abandoned)
+"da" = (
+/obj/structure/closet/emcloset{
```
Comment (ShizCalev) [inline _maps/shuttles/whiteship_delta.dmm]: /obj/structure/closet/emcloset/anchored

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Comment by Mark9013100** [comment]
Will fix, didn't notice the anchored version, thanks.

File: _maps/shuttles/whiteship_delta.dmm [hunk 6]
```diff
@@ -0,0 +1,2801 @@
...
+"cS" = (
+/obj/machinery/door/airlock/titanium{
+	cyclelinkeddir = 8;
+	name = "External Airlock"
+	},
+/obj/effect/decal/cleanable/dirt{
+	desc = "A thin layer of dust coating the floor.";
+	name = "dust"
+	},
+/obj/docking_port/mobile{
```
Comment (ShizCalev) [inline _maps/shuttles/whiteship_delta.dmm]: Need to set `timid = 1;` since this is a template version.

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: _maps/shuttles/whiteship_delta.dmm (hunk 6 above)
Comment (Mark9013100) [reply _maps/shuttles/whiteship_delta.dmm]: will do

Comment (Mark9013100) [reply _maps/shuttles/whiteship_delta.dmm]: should be timid 1 now

**Pull request merged**